### DISCUSSION TITLE
Istio Cross-Cluster Rollout

### DISCUSSION THREAD

**Discussion by reedjosh** [argoproj/argo-rollouts#3374] [General]
I have two meshed clusters. I would like to perform a rollout across the two -- potentially leveraging an external load balancer.

When I say across the two I mean I would like to move a workload from one cluster to the other in a staged manner.

I would prefer if this could be done at the admin level.

The intent is to be able to fire up a new cluster with modification (say a k8s upgrade), and then to be able to roll traffic and the underlying replicasets/pods to the new cluster with Istio meshing and Argo Rollouts.

Thanks!

**Comment by kostis-codefresh** [comment]
Currently not possible. Argo Rollouts only works in a single cluster.
